### ISSUE TITLE
你好，请问如何训练COCO数据集？

### ISSUE THREAD

**Issue by Caoxiaocao** [YunYang1994/tensorflow-yolov3#243]
我在你的文档里没有找到extract_coco.py   convert_tfrecord.py等文件，是已经删除了吗？

**Comment by zhangqinghao0811** [comment]
同问

**Comment by mumu0419** [comment]
同问

**Comment by xxllp** [comment]
+1

**Comment by ylqi007** [comment]
我把COCO的 train set 转换成 VOC 的格式了，然后可以训练，但是训练完后，COCO没有 2017 的 test set info，怎么测试？
> The test split don't have any annotations (only images).

@xxllp @mumu0419 @Caoxiaocao @zhangqinghao0811 你们有在COCO上训练吗，知道如何在COCO上测试吗？还请指教
